core/src/main/resources/jenkins/security/ApiTokenProperty/resources.js

```diff
@@ -25,34 +25,19 @@ Behaviour.specify(
   ".api-token-property-token-revoke",
   "api-token-property-token-revoke",
   0,
-  function (element) {
-    element.addEventListener("click", function (event) {
-      event.preventDefault();
-      revokeToken(element);
-    });
-  },
-);
-
-Behaviour.specify(
-  "#api-token-property-token-save",
-  "api-token-property-token-save",
-  0,
-  function (element) {
-    element.addEventListener("click", function () {
-      saveApiToken(element);
-    });
+  function (button) {
+    button.onclick = function () {
+      revokeToken(button);
+    };
   },
 );
 
 function revokeToken(anchorRevoke) {
-  var repeatedChunk = anchorRevoke.closest(".repeated-chunk");
-  var tokenList = repeatedChunk.closest(".token-list");
-  var confirmMessage = anchorRevoke.getAttribute("data-confirm");
-  var confirmTitle = anchorRevoke.getAttribute("data-confirm-title");
-  var targetUrl = anchorRevoke.getAttribute("data-target-url");
-
-  var inputUuid = repeatedChunk.querySelector("input.token-uuid-input");
-  var tokenUuid = inputUuid.value;
+  const tokenRow = anchorRevoke.closest(".token-card");
+  const confirmMessage = anchorRevoke.getAttribute("data-confirm");
+  const confirmTitle = anchorRevoke.getAttribute("data-confirm-title");
+  const targetUrl = anchorRevoke.getAttribute("data-target-url");
+  const tokenUuid = tokenRow.id;
 
   dialog
     .confirm(confirmTitle, { message: confirmMessage, type: "destructive" })
```

Comment:
Is there anyway to change 'Yes' to 'Revoke' in the dialog? A specific button title will help people understand the action they’re taking, without needing to read the rest of the dialog.

![revoke](https://github.com/user-attachments/assets/d86242de-46b0-4736-9075-f186f2a6ceee)

Reply:
done